code/game/turfs/simulated/wall_attacks.dm

```diff
@@ -130,9 +130,18 @@
 				for(var/obj/effect/overlay/wallrot/WR in src)
 					qdel(WR)
 				return
-		else if(!is_sharp(W) && W.force >= 10 || W.force >= 20)
-			to_chat(user, SPAN_NOTICE("\The [src] crumbles away under the force of your [W.name]."))
-			src.dismantle_wall(1)
+		else if(W.sharp)
+			user.visible_message("<b>[user]</b> starts scraping the rot away with \the [W].", SPAN_NOTICE("You start scraping the rot away with \the [W]."))
+			if(do_after(user, 3 SECONDS, TRUE))
```

Comment:
I'd consider making the time slightly random to simulate various levels of growth and whatnot.

Or perhaps it would even be possible to make it depend on how much rot (how many overlays) are there?